### ISSUE TITLE
[indent] decorators base indent

### ISSUE THREAD

**Issue by AjiTae** [typescript-eslint/typescript-eslint#1168]
**What code were you trying to parse?**

```ts
                   @Decorator()
class Foo {
}
```

**What did you expect to happen?**
Warning: Expected indentation of 0 spaces but found 10

**What actually happened?**
Nothing.

**Versions**

| package                                   | version |
| ----------------------------------------- | ------- |
| `@typescript-eslint/parser`               | `2.6.0` |
| `TypeScript`                              | `3.7.0-beta` |
| `ESLint`                                  | `6.6.0` |
| `node`                                    | `13.0.1` |

**Comment by lygstate** [comment]
Ping for this issue

**Comment by bradzacher** [comment]
@lygstate - commenting with a ping doesn't do anything except for annoy every single subscriber and watcher of this repo.

Your comment doesn't bump the issue or the PR up the queue.

Please do not spam everyone.

**Comment by lygstate** [comment]
@bradzacher Oh, I am trying to say I reproduced the problem, but after investigation, I found the problem are not reproduced now. I am going to a wrong direction

The version that not reproduce the problem.
**Versions**
 
package	version
`@typescript-eslint/parser`	`2.7.0`
`TypeScript`	`3.7.2`
`ESLint`	`6.6.0`
`node`	`12.4.0`
